pipelines/google_sheets_pipeline.py

```diff
@@ -1,13 +1,19 @@
 import dlt
-from google_sheets.google_sheets import google_spreadsheet
+from google_sheets import google_spreadsheet
 
-# constants
-SPREADSHEET_ID = ""
-SPREADSHEET_URL = ""
 
-# FULL PIPELINE RUN
-if __name__ == "main":
-    pipeline = dlt.pipeline(pipeline_name="google_sheets_pipeline", destination="bigquery", full_refresh=False, dataset_name="sample_google_sheet_data")
-    data = google_spreadsheet(spreadsheet_identifier=SPREADSHEET_ID)
+def full_pipeline_run() -> None:
```

Comment:
A more explanatory name would improve the readability for this module. `full_pipeline_run` is a bit too ambiguous. Consider using something similar to the chess example. For instance, `load_*`.